kickstarter/android-oss
# NT-1649: The user can resend the email from Interstitial

# 📲 What

When user taps resend email button on the email verification interstitial screen, a new verification email is sent to their inbox.

# 🛠 How

Use existing mutation to send a request to the backend to resend email. Display loading indicator upon button press that disappears once there is a response. 

# 👀 See

![Nov-25-2020 14-45-22](https://user-images.githubusercontent.com/19390326/100274697-e1e6d500-2f2c-11eb-981f-76c23aa20e4b.gif)

# 📋 QA

- Make sure you are signed in to your account and that you are on staging
- Tap the overflow menu in the top left corner of the app
- Tap "Internal Tools"
- Scroll to bottom of screen and tap "Email Verification Interstitial"
- Tap "Resend email" at the bottom of the screen
- Observe the loading indicator and success snack bar ("Verification email sent") 
- Put phone on airplane mode
- Tap "Resend email" again
- Observe loading indicator and error snack bar ("We couldn't resend this email - please try again")

# Story 📖

[NT-1649: The user can resend the email from Interstitial](https://kickstarter.atlassian.net/browse/NT-1649)
